spec/rubocop/cop/rails/has_many_or_has_one_dependent_spec.rb

```diff
@@ -40,6 +40,18 @@ class Person
         RUBY
       end
     end
+
+    context 'with_options dependent: :destroy' do
+      it "doesn't register an offense for `with_options dependent: :destroy`" do
```

Comment:
You basically repeated the same thing in the context and the example's description.

Reply:
Thanks for the review. I updated these with the following commit. How about this?
37f7d3c

Reply:
Yeah, that's much better.

Reply:
Thanks. I did squash.